spec/models/recipe_spec.rb

```diff
@@ -12,4 +12,24 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  describe 'total costs of ingedients method' do
```

Comment:
Great work adding this model test for the total_cost method! One thing that could make this test more robust would be adding an additional ingredient but *not* associating it to the recipe, in order to make sure that it does *not* get included in the total cost for that recipe